Gems/Atom/Tools/MaterialCanvas/Code/Source/MaterialCanvasApplication.cpp

```diff
@@ -43,7 +43,7 @@ namespace MaterialCanvas
         InitMaterialCanvasResources();
 
         QApplication::setOrganizationName("O3DE");
-        QApplication::setApplicationName("O3DE Material Canvas EXPERIMENTAL WIP");
+        QApplication::setApplicationName("O3DE Material Canva (Experimental) WIP");
```

Comment:
Small typo here

```suggestion
        QApplication::setApplicationName("O3DE Material Canvas (Experimental) WIP");
```

Also, do we need the `WIP` if we already have `(Experimental)` ?